### ISSUE TITLE
Chain false error on method import

### ISSUE THREAD

**Issue by david-zacharias** [wix/eslint-plugin-lodash#186]
The project I am contributing to uses import scope member (see Example 2 below). I wanted to  enforce lodash, but that leads to issue with existing chain syntax, marking these invalid. 

**Example 1: Passing**
```js
import _ from 'lodash';

_.chain(['a', 'b', 'c'])
  .map(_.toUpper)
  .reverse()
  .value();
```

**Example 2: Failing**
```js
import { chain, toUpper } from 'lodash';

chain(['a', 'b', 'c']) // error  Prefer '_.map' over the native function  lodash/prefer-lodash-method
  .map(toUpper)
  .reverse()
  .value();
```

Is this expected? Thanks for feedback in advance!

**Comment by ganimomer** [comment]
Hi, and thanks for using our plugin!
I wasn't aware of the case of using `chain` but still importing the full Lodash.
Yes, it looks like an invalid case for the rule to report on. I'll look into the PR.

### Side note:
It looks like your second example would work as expected, but this would not:

```js
import chain from 'lodash/chain'
import toUpper from 'lodash/toUpper'

chain(['a', 'b', 'c']) // error  Prefer '_.map' over the native function  lodash/prefer-lodash-method
  .map(toUpper)
  .reverse()
  .value();
```
You should note that it looks like importing `chain` like that has a penalty of importing full Lodash.
